Documentation/Global/Form/Upload.rst

```diff
@@ -17,14 +17,37 @@ Form.upload ViewHelper `<f:form.upload>`
 Examples
 ========
 
-Example::
+
+..  _typo3-fluid-form-upload-example-basic:
+
+Basic usage
+-----------
+
+..  code-block:: html
+    :caption: EXT:my_extension/Resources/Private/Partials/Upload.html
 
    <f:form.upload name="file" />
 
-Output::
+..  code-block:: html
+    :caption: HTML output
 
    <input type="file" name="file" />
 
+..  _typo3-fluid-form-upload-example-accept:
+
+Example for allowed MIME types / file extensions (native HTML5 attribute)
+-------------------------------------------------------------------------
+
+..  code-block:: html
+    :caption: EXT:my_extension/Resources/Private/Partials/Upload.html
+
+   <f:form.upload name="file" accept=".gif,.jpg,image/jpg,image/jpeg,image/gif" />
+
+..  code-block:: html
+    :caption: HTML output
+
+   <input type="file" name="file" accept=".gif,.jpg,image/jpg,image/jpeg,image/gif" />
+
```

Comment:
```suggestion
..  hint::

    Please be aware that the `accept` attribute is a native HTML5 element. It has no specific Extbase or Fluid implementation, so validation of the incoming file is only performed in the client. It does not replace proper validation on the backend / server side, so proper `UploadedFile` or Extbase File-Upload-Handling validation needs to be implemented, too.
```

Reply:
Maybe formulate more direct, like

The `accept`  attribute is a native HTML5 element. It has no specific Extbase or Fluid implementation, and validation of the incoming file is only performed on the client. It does not replace proper server-side validation, so UploadedFile or Extbase file-upload handling must still be implemented.